### DISCUSSION TITLE
Google AUTH

### DISCUSSION THREAD

**Discussion by abhinavgautam07** [provectus/kafka-ui#3167] [Q&A] [answered]
How to setup google auth when orchestration UI using helm or kubernetes.

**Comment by Haarolean** [comment, marked as answer]
OAuth2 setup examples are [here](https://github.com/provectus/kafka-ui/wiki/OAuth-Configuration)
Helm documentation is [here](https://github.com/provectus/kafka-ui/blob/master/charts/kafka-ui/README.md)

Helm charts support passing configs either by env variables (docker-compose way) or yaml config.

**Comment by abhinavgautam07** [comment]
facing the error invalid client-registration-id

**Reply by Haarolean** [reply]
Might be #3154. Try pulling `master` image.
